scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -148,12 +144,18 @@ public void doTask(ExecutableContainer executableContainer, TaskResult[] previou
             progressFileReader.start(dataspaces.getScratchFolder(), taskId);
 
             TaskContext context = new TaskContext(executableContainer, initializer, previousTasksResults,
-                dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
-                dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
-                        .toString(), getHostname());
+                    dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
+                    dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
+                    .toString(), getHostname());
 
             File workingDir = getTaskWorkingDir(context, dataspaces);
 
+            logger.info("Task working dir : " + workingDir);
```

Comment:
Yes, logback + slf4j will do the job when we will upgrade the libs. We have only 400 debug logging statements in the code, should take way less than 3 days to change. 
In the meantime I would say it's fine to leave DEBUG level logging without isDebugEnabled unless it is done to avoid really expensive operations.